fe/fe-core/src/main/java/com/starrocks/catalog/SchemaTable.java

```diff
@@ -523,6 +523,16 @@ public static Builder builder() {
                                     .column("STATE", ScalarType.createVarchar(NAME_CHAR_LEN))
                                     .column("TYPE", ScalarType.createVarchar(NAME_CHAR_LEN))
                                     .build()))
+                    .put("be_metrics", new SchemaTable(
```

Comment:
How can you fetch metrics from all backends? Seem this only chooses one of all backends.

Reply:
if the query doesn't have a filter like `be_id = xx`, the planer will assign a scanrange for each alive BE. If the query does have a fliter like `be_id = x`, the planner will assign a single scan range at that BE.